scss/starter.scss

```diff
@@ -22,26 +22,22 @@
 
 // Option 2
 //
-// Import just the styles you need. Note that some stylesheets are required no matter what.
-
-@import "bootstrap/scss/functions"; // Required
-@import "bootstrap/scss/variables"; // Required
-@import "bootstrap/scss/mixins"; // Required
-
-//
-// Override Bootstrap here
-//
+// Place variable overrides first, then import just the styles you need. Note that some stylesheets are required no matter what.
 
 // Toggle global options
 $enable-gradients: true;
 $enable-shadows: true;
 
 // Customize some defaults
-$body-color: $purple;
+$body-color: #333;
 $body-bg: #f5f5f5;
 $border-radius: .4rem;
 
-// @import "bootstrap/scss/root";
+@import "bootstrap/scss/functions"; // Required
+@import "bootstrap/scss/variables"; // Required
```

Comment:
users cant use functions to set variables,  `_functions.scss` must be pure - variables as default arguments is easy but not right

Reply:
I guess functions should be included at top, otherwise we can't use functions like `color($gray)` `theme-color($dark)` etc

when i tried without including functions at top, output contains raw code like
`color: color("dark")` and when i include functions before custom variables, output is being parsed properly.

Reply:
now i come across another issue.... since we are not importing `_variables`, we can't use all variables, we can only use variables which is modified
ex: if i didn't add `$colors(dark:#xxx)` then i can't use `color("dark")`

so only option seems to be adding all original variables including maps but that causes a minute issue....

lets say i added `$colors(blue:$blue)` then in the output css file, in `:root()` i see blue color two times... 

it is like a catch-22 situation.

but i guess that is an issue with sass itself, not with bootstrap.

Reply:
@lorvent you can use [utilities/api](https://getbootstrap.com/docs/5.0/utilities/api/)

Reply:
@WinterSilence it is not available for bootstrap 4

Reply:
@WinterSilence i want to have a different color of headings so trying to override `$headings-color` instead of applying utility classes everytime..